loader/CMakeLists.txt

```diff
@@ -3,6 +3,11 @@
 # "tests" variable
 ign_get_libsources_and_unittests(sources tests)
 
+# Disable ign_TEST if ignition-tools is not found
+if (MSVC OR NOT IGNITION-TOOLS_BINARY_DIRS)

```

Comment:
Should we open a ticket to track command line support on Windows?

Reply:
I think https://github.com/ignitionrobotics/ign-tools/issues/7 will help with windows support eventually